### ISSUE TITLE
runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by will** [divnix/std#282]
It seems to crash when you tab over to a thing with multiple items and then go down.

on the std-book 09bfd0761bf7f8726807996e6de786040826e209
std version 0.20.0 (dirty)

https://user-images.githubusercontent.com/1973/226871276-9cc5347c-9f03-4c89-923f-9a43388dff09.mov

```
Caught panic:

runtime error: invalid memory address or nil pointer dereference

Restoring terminal...

goroutine 1 [running]:
runtime/debug.Stack()
	runtime/debug/stack.go:24 +0x64
runtime/debug.PrintStack()
	runtime/debug/stack.go:16 +0x1c
github.com/charmbracelet/bubbletea.(*Program).StartReturningModel.func3()
	github.com/charmbracelet/bubbletea@v0.22.0/tea.go:371 +0x8c
panic({0x101816fe0, 0x101bfa340})
	runtime/panic.go:890 +0x258
github.com/paisano-nix/paisano/flake.GetActionEvalCmdArgs({0x14000020940, 0x8}, {0x140000209b0, 0x7}, {0x140000209d0, 0x9}, {0x140000209c0, 0x7}, 0x0)
	github.com/paisano-nix/paisano/flake/flake.go:127 +0x70
main.(*Tui).GetActionCmd(0x101?, 0x1015d2c7c?)
	github.com/paisano-nix/paisano/tui.go:171 +0x49c
main.(*Tui).SetInspect(0x1400042ca00)
	github.com/paisano-nix/paisano/tui.go:152 +0x90
main.(*Tui).Update(0x1400042ca00, {0x101840e20?, 0x140006bf920?})
	github.com/paisano-nix/paisano/tui.go:398 +0xf24
github.com/charmbracelet/bubbletea.(*Program).StartReturningModel(0x1400019e370)
	github.com/charmbracelet/bubbletea@v0.22.0/tea.go:540 +0xf88
main.main()
	github.com/paisano-nix/paisano/main.go:52 +0x64
panic: interface conversion: tea.Model is nil, not *main.Tui

goroutine 1 [running]:
main.main()
	github.com/paisano-nix/paisano/main.go:54 +0x158
```

**Comment by blaggacao** [comment]
Thanks for the report, I've been marginally aware, already, so this is just the right nudge to fix it! Thanks! :smile: 

As a bug I'll prioritise.

**Comment by blaggacao** [comment]
Hey @will , a fix has landed on  `main` or also `v0.20.2`. Thanks for the report.

**Comment by will** [comment]
Works great thanks!
